src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTaggerProvider.cs

```diff
@@ -16,25 +18,34 @@ namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
     /// The provider that is used as a middleman to create the tagger so that the data tag
     /// can be used to create the UI tag
     /// </summary>
-    [Export(typeof(ITaggerProvider))]
+    [Export(typeof(IViewTaggerProvider))]
     [ContentType(ContentTypeNames.RoslynContentType)]
     [TagType(typeof(IntraTextAdornmentTag))]
     [Name(nameof(InlineParameterNameHintsTaggerProvider))]
-    internal class InlineParameterNameHintsTaggerProvider : ITaggerProvider
+    internal class InlineParameterNameHintsTaggerProvider : IViewTaggerProvider
     {
         private readonly IBufferTagAggregatorFactoryService _bufferTagAggregatorFactoryService;
+        public readonly IClassificationFormatMapService ClassificationFormatMapService;
+        public readonly IClassificationTypeRegistryService ClassificationTypeRegistryService;
+        public readonly IThreadingContext ThreadingContext;
```

Comment:
```suggestion
        public IClassificationFormatMapService ClassificationFormatMapService { get; }
        public IClassificationTypeRegistryService ClassificationTypeRegistryService { get; }
        public IThreadingContext ThreadingContext { get; }
```

ie, public fields are usually avoided. Auto-properties without a setter will only be able to be set in the constructor, just like readonly fields.